### PR TITLE
Add examples for uplink tunnel ingress

### PR DESCRIPTION
**Pull request by welteki**
Signed-off-by: Han Verstraete (OpenFaaS Ltd) <han@openfaas.com>

## Description

Add examples for uplink tunnel ingress.

## Motivation and Context

Show users how to configure inlets for tunnels with inlets uplink.

## How Has This Been Tested?

Verified site locally. Verified instructions and kubernetes resources.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`

### CODE REVIEW

File: docs/uplink/ingress-for-tunnels.md [hunk 1]
```diff
@@ -0,0 +1,275 @@
+# Ingress for tunnels
+
+Make inlets uplink HTTP tunnels publicly accessible by setting up ingress for the data plane.
+
+The instructions assume that you want to expose two HTTP tunnels. We will configure ingress for the first tunnel, called `grafana`, on the domain `grafana.example.com`. The second tunnel, called `openfaas`, will use the domain `openfaas.example.com`.
+
+Both tunnels can be created with `kubctl` or the `inlet-pro` cli. See [create tunnels](/uplink/create-tunnels/) for more info:
```
Comment (alexellis) [inline docs/uplink/ingress-for-tunnels.md]: kub`e`ctl
inlet`s`-pro

File: docs/uplink/ingress-for-tunnels.md [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+            port:
+              number: 8000
+  tls:
+  - hosts:
+    - openfaas.example.com
+    secretName: openfaas-cert
+```
+
+
+## Setup tunnel ingress with Istio Ingress gateways
```
Comment (alexellis) [inline docs/uplink/ingress-for-tunnels.md]: `Ingress gateways` or `an Ingress gateway` ?

File: docs/uplink/ingress-for-tunnels.md [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+        tls:
+          mode: SIMPLE
+          credentialName: openfaas-cert
+        hosts:
+        - openfaas.example.com
+    ```
+
+    Note that the `credentialsName` references the secrets for the certificates created in the previous step.
+
+4. Configure the gateway’s traffic routes by defining corresponding virtual services:
```
Comment (alexellis) [inline docs/uplink/ingress-for-tunnels.md]: `’` => `'`

File: docs/uplink/ingress-for-tunnels.md [hunk 4]
```diff
@@ -0,0 +1,275 @@
+# Ingress for tunnels
+
```
Comment (alexellis) [inline docs/uplink/ingress-for-tunnels.md:2]: Info message (look up the syntax)

https://squidfunk.github.io/mkdocs-material/reference/admonitions/

Inlets Uplink is designed to connect customer services to a remote Kubernetes cluster for command and control as part of a SaaS product.

Any tunnelled service can be accessed directly from within the cluster and does not need to be exposed to the public Internet for access.

Beware: by following these instructions, you are exposing one or more of those tunnels to the public Internet.

**Review by alexellis — COMMENTED** [review]
Please see typos, and add more context in the intro.

**Comment by welteki** [comment]
The PR has been updated with the requested changes.

**Review by alexellis — APPROVED** [review]
Approved

**Pull request merged**